third_party/amd/backend/compiler.py

```diff
@@ -270,6 +270,22 @@ def make_ttgir(mod, metadata, options):
         pm.run(mod)
         return mod
 
+    @staticmethod
```

Comment:
let's skip amd for now as gluon doesn't support it yet

Reply:
I don't see why not? The copy kernel is running on amd in CI already.

Reply:
I'm fine keeping it if you think it's not a distraction